src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -36,9 +56,32 @@ internal sealed partial class NullableWalker : DataFlowPassBase<NullableWalker.L
 
         // PROTOTYPE(NullableReferenceTypes): Remove the Binder if possible. 
         private readonly Binder _binder;
-
         private readonly Conversions _conversions;
 
+        [Flags]
+        private enum MethodSignatureUse
+        {
+            None = 0,
+            ReturnType = 1,
+            ParameterTypes = 2,
+        }
+
+        /// <summary>
+        /// Which fields to use from _methodSignatureOpt.
+        /// </summary>
```

Comment:
To use for what? And if you're not using the parameter types, what would you be using instead?